A/autossh/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "autossh"
+version = v"1.4.0" # autossh v1.4f, adapted to the BinaryBuilder requirements
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/Autossh/autossh/releases/download/v1.4f/autossh-1.4f.tgz", "0172e5e1bea40c642e0ef025334be3aadd4ff3b4d62c0b177ed88a8384e2f8f2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd autossh-1.4f/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-ssh=${prefix}/bin \
+            ac_cv_func_malloc_0_nonnull=yes ac_cv_func_realloc_0_nonnull=yes
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl")
+]
+
```

Comment:
```suggestion
platforms = supported_platforms(; exclude=!Sys.islinux)
```
What's the problem with the non-linux platforms?

Reply:
The original package is intended for only Linux and MacOS.

Thanks for the review!